templates/server/index.html

```diff
@@ -365,156 +423,261 @@ <h2>Security support for Ubuntu releases</h2>
-            <hr class="p-rule" />
-            <a href="https://maas.io/">Learn more about MAAS&nbsp;&rsaquo;</a>
+        </div>
+        <div id="ubuntu-pro-new-mexico"
+             class="p-tabs__content"
+             role="tabpanel"
+             aria-labelledby="ubuntu-pro-tab-new-mexico">
+          <div class="p-strip is-shallow u-no-padding--top">
+            <div class="p-strip is-shallow">
+              {{ image(url="https://assets.ubuntu.com/v1/b0ee8fee-nmsu-logo.svg",
+              alt="New Mexico State University",
+              width="45",
+              height="50",
+              hi_def=True,
+              fmt="svg",
+              loading="lazy"
+              ) | safe
+              }}
+            </div>
+            <h3>How New Mexico State University accelerates compliant federal research with Ubuntu</h3>
+            <p>When the stakes are high and national security is on the line, every decision matters. Just ask the team at New Mexico State University's Physical Science Laboratory (PSL).</p>
+            <hr class="p-rule--muted" />
+            <a href="/blog/how-new-mexico-state-university-accelerates-compliant-federal-research-with-ubuntu" aria-label="Read the case study about New Mexico State University">Read the case study&nbsp;&rsaquo;</a>
           </div>
         </div>
       </div>
     </div>
-    <div class="row p-section--shallow">
-      <div class="col-9 col-medium-5 col-start-large-4 col-start-medium-2">
-        <hr class="p-rule" />
-        <div class="row">
-          <div class="col-3 col-medium-2">
-            {{ image(url="https://assets.ubuntu.com/v1/e3805af7-lxd-tagged-logo.png",
-                        alt="LXD",
-                        width="61",
-                        height="32",
-                        hi_def=False,
-                        loading="lazy") | safe
-            }}
-          </div>
-          <div class="col-6 col-medium-3">
-            <p class="p-heading--5">Machine containers</p>
+  </section>
+
+  {% call(slot) vf_equal_heights(
+      title_text="Use it with other Canonical products",
+      highlight_images=False,
+      image_aspect_ratio_large="16-9",
+      items=[
+      {
+      "title_text": "Multi-cloud orchestration",
+      "image_html":  image(url="https://assets.ubuntu.com/v1/11b77504-multi_cloud.svg",
+            alt="Canonical Juju",
+            width="284",
+            height="161",
+            hi_def=True,
+            loading="auto|lazy",
+            fmt="svg",
+            attrs={"class": "p-image-container__image"}) | safe,
+      "description_html": "<p>With the option of a command line or browser-based interface, Juju enables you to design and deploy entire workloads in just a few clicks. It works on public clouds like AWS and Microsoft Azure, private clouds built on OpenStack and even directly on bare metal, via MAAS.</p>",
+      "cta_html": "<a href='https://jujucharms.com/'>Learn more about Juju&nbsp;&rsaquo;</a>"
```

Comment:
Since this redirects to https://charmhub.io/ we can put it here directly
```suggestion
      "cta_html": "<a href='https://charmhub.io/'>Learn more about Juju&nbsp;&rsaquo;</a>"

```